integration_tests/family/test_create.py

```diff
@@ -24,28 +25,47 @@ def test_create_family(client: TestClient, data_db: Session, user_header_token):
         "/api/v1/families", json=new_family.model_dump(), headers=user_header_token
     )
     assert response.status_code == status.HTTP_201_CREATED
-    import_id = response.json()
-    assert import_id == "CCLW.family.i00000001.n0000"
-    actual_family = data_db.query(Family).filter(Family.import_id == import_id).one()
+    expected_import_id = "CCLW.family.i00000001.n0000"
+    assert response.json() == expected_import_id
+    actual_family = (
+        data_db.query(Family).filter(Family.import_id == expected_import_id).one()
+    )
 
     assert actual_family.title == "Title"
```

Comment:
Do we also want to test the `actual_family.organisation_id` here too - just to ensure its still being used'?

Reply:
Good shout